INSTALL.md

```diff
@@ -1,25 +1,27 @@
 ## Installation Steps
 
-Helm charts are provided inside https://github.com/eclipse-tractusx/sd-factory
-
-1.) Using helm commands: <br />
+Helm chart is provided inside https://github.com/eclipse-tractusx/sd-factory
 
 How to install application using helm:  <br />
-    helm install ReleaseName ChartName
+
+              helm install [ReleaseName] [ChartName]
+
+1.) Installation from released chart: <br />
     
-    a.) Add helm repository in tractusx:
-           helm repo add sd-factory https://eclipse-tractusx.github.io/charts/dev
+    a.) Add helm repository for tractusx-dev:
+           helm repo add tractusx-dev https://eclipse-tractusx.github.io/charts/dev
     b.) To search the specific repo in helm repositories 
-           helm search repo sd-factory/sdfactory
-    c.) To install using helm command:
-           helm install sdf sd-factory/sdfactory
+           helm search repo tractusx-dev/sdfactory
+    c.) To set your own configuration and secret values, install the helm chart with your own values file in kubernetes cluster:
+           helm install -f your-values.yaml [ReleaseName] tractusx-dev/sdfactory -n [NameSpace]
 
 
-2.) Local installation:
-
+2.) Installation from repository:
+        
     a.) git clone https://github.com/eclipse-tractusx/sd-factory.git
-    b.) Modify values file according to your requirement
-    c.) You need to define the secrets as well in values.yaml
+    b.) cd sd-factory
+    c.) Modify values file according to your requirement
+    d.) You need to define the secrets as well in your-values.yaml. These secrets should be defined in Hashicorp vault to keep them secure
```

Comment:
The whole part regarding "These secrets should be defined in Hashicorp vault ..." would be more appropriate after 1.c. Then just remove 2.c and 2.d. (they are redundant) and change 2.e to 2.c.

Reply:
Modified the statements